Open Judge System/Web/OJS.Web/Scripts/Administration/Problems/problems-create.js

```diff
@@ -12,6 +12,35 @@ function selectedFile(e) {
     $('#file-button-' + id).text(fileName.length > 20 ? fileName.substring(0, 20) + '...' : fileName);
 }
 
+function validateFile(files, validationBox) {
+    if (files) {
+        if (files[0].extension.toLowerCase() !== ".zip") {
+            validationBox.attr("class", "field-validation-error");
+            validationBox.text("Uploaded file must be in .zip format");
+            e.preventDefault();
+        } else {
+            validationBox.attr("class", "field-validation-valid");
+            validationBox.text("");
+        }
+    }
+
+    $("#create-form form").removeData("validator");
+    $("#create-form  form").removeData("unobtrusiveValidation");
+    $.validator.unobtrusive.parse($('#create-form form'));
```

Comment:
Please be consistent with the type of quotes in JavaScript. Use only `'` or only `"`, but not both.